A semicolon missing in the Maemo 5 branch (Q_WS_MAEMO_5, which Leste's maemo qmake spec defines).

--- a/src/mainform.cpp
+++ b/src/mainform.cpp
@@ -137,7 +137,7 @@
 #ifdef Q_WS_MAEMO_5
         //setAttribute(Qt::WA_Maemo5PortraitOrientation, true);
         //setAttribute(Qt::WA_Maemo5LandscapeOrientation, false);
-        setProperty("X-Maemo-Orientation", 1)
+        setProperty("X-Maemo-Orientation", 1);
 #endif
     }
 }
